pyproject.toml

```
@@ -13,7 +13,7 @@ maintainers = [
 requires-python = ">=3.8"
 dependencies = [
     'asciitree',
-    'numpy>=1.7',
+    'numpy>=1.17',
```

Actually we are suppose to be requiring NumPy 1.20 after PR ( https://github.com/zarr-developers/zarr-python/pull/988 ). This must be a holdover. Could we please bump this to 1.20?

```suggestion
    'numpy>=1.20',
```